portfolio/src/main/webapp/script.js

```diff
@@ -12,11 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/*
+/**
```

Comment:
The one * was actually the right way to go. https://google.github.io/styleguide/jsguide.html#formatting-block-comment-style